iceoryx_hoofs/include/iceoryx_hoofs/internal/cxx/storable_function.hpp

```diff
@@ -48,14 +48,13 @@ class storable_function;
 ///                     See static_storage.hpp for a static memory version.
 /// @tparam ReturnType  The return type of the stored callable.
 /// @tparam Args        The arguments of the stored callable.
-template <typename StorageType, typename ReturnType, typename... Args>
-class storable_function<StorageType, signature<ReturnType, Args...>>
+template <uint64_t Capacity, typename ReturnType, typename... Args>
+class storable_function<Capacity, signature<ReturnType, Args...>>
```

Comment:
Please update the documentation.

Reply:
Please update the whole class documentation. It still describes that the storage type is exchangeable.